src/Umbraco.Cms.Api.Management/Controllers/User/ChangePasswordUserController.cs

```diff
@@ -17,22 +22,37 @@ public class ChangePasswordUserController : UserControllerBase
     private readonly IUserService _userService;
     private readonly IUmbracoMapper _mapper;
     private readonly IBackOfficeSecurityAccessor _backOfficeSecurityAccessor;
+    private readonly IAuthorizationService _authorizationService;
 
     public ChangePasswordUserController(
         IUserService userService,
         IUmbracoMapper mapper,
-        IBackOfficeSecurityAccessor backOfficeSecurityAccessor)
+        IBackOfficeSecurityAccessor backOfficeSecurityAccessor,
+        IAuthorizationService authorizationService)
     {
         _userService = userService;
         _mapper = mapper;
         _backOfficeSecurityAccessor = backOfficeSecurityAccessor;
+        _authorizationService = authorizationService;
     }
 
-    [HttpPost("change-password/{id:guid}")]
+    [HttpPost("{id:guid}/change-password")]
     [MapToApiVersion("1.0")]
-    [ProducesErrorResponseType(typeof(ChangePasswordUserResponseModel))]
+    [ProducesResponseType(typeof(ChangePasswordUserResponseModel), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
     public async Task<IActionResult> ChangePassword(Guid id, ChangePasswordUserRequestModel model)
     {
+        AuthorizationResult authorizationResult = await _authorizationService.AuthorizeResourceAsync(
+            User,
+            UserPermissionResource.WithKeys(id),
+            AuthorizationPolicies.AdminUserEditsRequireAdmin);
```

Comment:
This block of code seems to be duplicated through a lot of user controllers. Consider refactoring - for example, find inspiration in `UpdateDocumentControllerBase.HandleRequest`

Reply:
I considered this, but since we will most likely be changing these policies with policies that use granular permissions in the (hopefully not to distant) future. I find it wise to keep it as is for now.